data_transformer/main.py

```diff
@@ -1,18 +1,36 @@
-from data_saver import get_exchange_rate
-from common import notify_tg
-from data_to_sheet import write_data
+import datetime
+from flask import Flask, request
+from flask import typing as flask_typing
 
+from data_transformer import transform
 
-target_note = get_exchange_rate(bank='mono')
-notify_tg(target_note)
+app = Flask(__name__)
 
-write_data(target_note)
-notify_tg('it`s done 👌')
 
+@app.route('/', methods=['POST'])
+def main() -> flask_typing.ResponseReturnValue:
+    """
+    Controller that accepts command via HTTP and
+    trigger business logic layer: Transforming data
+    """
 
-def main():
-    pass
+    input_data: dict = request.json
+
+    if "raw_dir" not in input_data or "stg_dir" not in input_data:
+        return {
+                   "message": "No path received. 'raw_dir' and 'stg_dir' need to be specified",
+               }, 400
+    else:
+        raw_dir_path = f"{input_data['raw_dir']}"
+        stg_dir_path = f"{input_data['stg_dir']}"
```

Comment:
Why do you f-stringify those 2 parameters? They're strings already.
You can just use:
```
raw_dir_path = finput_data['raw_dir']
stg_dir_path = input_data['stg_dir']
```